src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react'
+import {useAuth0} from '@auth0/auth0-react'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import CardContent from '@mui/material/CardContent'
+import CardHeader from '@mui/material/CardHeader'
+import InputBase from '@mui/material/InputBase'
+import Paper from '@mui/material/Paper'
+import useTheme from '@mui/styles/useTheme'
+import {TooltipIconButton} from '../Buttons'
+import useStore from '../../store/useStore'
+import {postIssue} from '../../utils/GitHub'
+import Submit from '../../assets/icons/Submit.svg'
+
+
+/**
+ * Note card create
+ *
+ * @param {string} username
+ * @param {string} avatarUrl
+ * @return {object} React component
```

Comment:
fyi, this is a correct return type, but for function components `@return {React.Component}` is more specific and I've been moving to that around the codebase